src/sentry/api/base.py

```diff
@@ -103,7 +103,7 @@ class Endpoint(APIView):
     # Default Rate Limit Values, override in subclass
     # Should be of format: { <http function>: { <category>: RateLimit(limit, window) } }
     rate_limits: Mapping[str, Mapping[RateLimitCategory | str, RateLimit]] = {}
-    enforce_rate_limit: bool = settings.SENTRY_RATELIMITER_ENABLED
+    enforce_rate_limit: bool = False
```

Comment:
is this still used?

Reply:
Not right now, but I'm planning on deleting the `rate_limit_endpoint` decorator and using this instead as a sort of soft rollout.